__tests__/Getter.test.js

```diff
@@ -0,0 +1,19 @@
+import { getter } from '../src/Getter'
+import { toArray } from '../src/operations'
+
+const obj = {
+  foo: [1, 2, 3],
+  bar: 'baz',
+}
+
+describe('Getter', () => {
+  test.skip('asFold', () => {
+    const fold = getter('foo').asFold
+    expect(toArray(fold, obj)).toEqual([1, 2, 3])
```

Comment:
@serras how can I tests this correctly? Missing tests for `Getter` include `asFold` and `asPartialGetter` 😭

Reply:
`getter(foo)` does not work, you need to spell out the whole thing `getter((x) => x.foo)`.

I usually do a bunch of tests like the following, until I cover every case:

```js
const g = getter((x) => x.foo)
expect(g.toArray(obj)).toEqual([1, 2, 3])
expect(toArray(g.asPartialGetter, obj)).toEqual([1, 2, 3])
expect(toArray(g, obj)).toEqual([1, 2, 3])
```

Reply:
awesome thanks!